src/components/selections/attributes.js

```diff
@@ -54,6 +54,51 @@ module.exports = overrideAll(templatedArray('selection', {
         description: 'Sets the selection\'s end y position.'
     },
 
+    x0shift: {
+        valType: 'number',
+        dflt: 0,
+        min: -1,
+        max: 1,
```

Comment:
Let's mention in the descriptions that *-0.5* is the start of the category and *0.5* is the end of it.

Reply:
done